packages/core/react/src/WalletProvider.tsx

```diff
@@ -132,23 +132,22 @@ export const WalletProvider: FC<WalletProviderProps> = ({
     const select = useCallback(
         async (walletName: WalletName | null) => {
             if (name === walletName) return;
-            if (adapter) await adapter.disconnect();
```

Comment:
This seems to be missing the adapter disconnect effect from #212

Reply:
Bah! Unintentional commit.